flint.ui/src/main.js

```diff
@@ -37,6 +37,10 @@ import VueMaterial from 'vue-material'
 // Vue.use(MdList)
 // Vue.use(MdDialog)
 // Vue.use(MdContent)
+
+// import VueUploadComponent from 'vue-upload-component'
```

Comment:
Let's delete this if it's not going to be used. Otherwise it accumulates in the code base and noone remembers why it's there.